test/e2e/testsuite/testsuite.go

```diff
@@ -1004,10 +1002,6 @@ func setPodmanEnv() error {
 }
 
 func SetConfigPropertyToValueSucceedsOrFails(property string, value string, expected string) error {
-	// Since network-mode is only supported on Linux, we skip this property test for non-linux platforms
-	if property == "network-mode" && runtime.GOOS != "linux" {
-		return nil
-	}
```

Comment:
Shouldn't this still true? On linux network-mode is still have option for `system` and `user` (being default)?